src/lib/components/namespace-list.svelte

```diff
@@ -74,7 +77,11 @@
           class="first:rounded-t-xl first:border-t-3 last:rounded-b-xl last:border-b-3 border-b border-l-3 border-r-3 border-gray-900 flex border-collapse gap-2 hover:bg-gradient-to-br from-blue-100 to-purple-100 cursor-pointer"
         >
           <a
-            href={namespace.href(namespace.namespace)}
+            href={routeForWorkflowsWithQuery({
```

Comment:
I've opted to add the query here instead of doing ` updateQueryParamsFromFilter` on a namespace change as `$page.params.namespace` is getting updated quite a bit and it felt excessive to be constantly updating the query params all the time.

Reply:
Yup agree this is the right move.